pkg/metrics/metrics.go

```diff
@@ -969,6 +987,39 @@ func CreateConfiguration(metricsEnabled []string) (Configuration, []prometheus.C
 
 			collectors = append(collectors, BPFMapOps)
 			c.BPFMapOps = true
+
+		case Namespace + "_triggers_policy_update_total":
+			TriggerPolicyUpdateTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+				Namespace: Namespace,
+				Subsystem: "triggers",
+				Name:      "policy_update_total",
+				Help:      "Total number of policy update trigger invocations labelled by reason",
+			}, []string{"reason"})
+
+			collectors = append(collectors, TriggerPolicyUpdateTotal)
+			c.TriggerPolicyUpdateTotal = true
+
+		case Namespace + "_triggers_policy_update_folds":
+			TriggerPolicyUpdateFolds = prometheus.NewGauge(prometheus.GaugeOpts{
+				Namespace: Namespace,
+				Subsystem: "triggers",
+				Name:      "policy_update_folds",
+				Help:      "Current number of folds",
+			})
+
+			collectors = append(collectors, TriggerPolicyUpdateFolds)
+			c.TriggerPolicyUpdateFolds = true
+
+		case Namespace + "_triggers_policy_update_call_duration_seconds":
```

Comment:
Could make "triggers" a const string, then the name of each metric a const string and be able to use them in the case statements + the "Subsystem" and "Name" fields for each trigger to reduce duplicate hardcoded strings.